internal/host/validator.go

```diff
@@ -472,7 +472,8 @@ func (v *validator) printAPIVipConnected(c *validationContext, status validation
 }
 
 func (v *validator) belongsToMajorityGroup(c *validationContext) validationStatus {
-	if IsDay2Host(c.host) {
+	// TODO IPv6 Change
```

Comment:
Why is there a todo here?

Reply:
This is only phase 2.  Connectivity check validation is not yet supported. On phase 3 it should be changed

Reply:
Then please write a proper comment.